tests/test_log_artifact.py

```diff
@@ -44,3 +45,61 @@ def test_log_artifact_with_save_dvc_exp(tmp_dir, mocker, mocked_dvc_repo):
         include_untracked=[live.dir, "data", ".gitignore"],
         force=True,
     )
+
+
+def test_log_artifact_type_model(tmp_dir, mocked_dvc_repo):
+    (tmp_dir / "model.pth").touch()
+
+    with Live() as live:
+        live.log_artifact("model.pth", type="model")
+
+    assert load_yaml(live.dvc_file) == {
+        "artifacts": {"model": {"path": "model.pth", "type": "model"}}
```

Comment:
TODO: Fix the `path` here.
It should be written as relative to the `dvclive/dvc.yaml`

Reply:
I pushed a fix